lib/formConfigTransformer.ts

```
@@ -0,0 +1,287 @@
+
+    // Map field keys to user data properties
+    switch (fieldKey) {
+      case 'firstName':
+        value = String(user.firstName || '');
+        break;
+      case 'lastName':
+        value = String(user.lastName || '');
+        break;
+      case 'name':
+        // Handle both single name field and separate firstName/lastName fields
+        const firstName = String(user.firstName || '');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)